docs/notes/2.23.x.md

```diff
@@ -61,6 +61,10 @@ Docker inference is improved. Pants can now make inferences by target address fo
 
 Experimental support for a rust-based dockerfile parser can be enabled via `[dockerfile-parser].use_rust_parser` option.
 
+#### JVM
+
+When `jvm.reproducible_jars` flag is set resources jars would also be made reproducible, previously it was assumed resources jars are reproducible without additional action.
```

Comment:
Just some minor copy editing, plus including a link since it's possible a reader is discovering the `reproducible_jars` options for the first time by reading these and may be interested to discover more and enable it for themselves!

```suggestion
When [the `jvm.reproducible_jars` flag](https://www.pantsbuild.org/2.21/reference/subsystems/jvm#reproducible_jars) is set resources jars are now also made reproducible, previously it was assumed resources jars are reproducible without additional action.
```